app/sanity/queries/categories.ts

```diff
@@ -0,0 +1,72 @@
+import groq from "groq"
+import type { SanityStegaClient } from "@sanity/client/stega"
+import { LocalizedString } from "~/sanity/queries/shared"
+
+export type Category = {
+  name: LocalizedString
+  description: LocalizedString
+}
+
+export const categoriesQuery = groq`*[_type == "categoryType"] | order(_createdAt desc) {
+  _id,
+  "name": {
+    "en": name.en,
+    "fr": name.fr,
+  },
+  "description": {
+    "en": description.en,
+    "fr": description.fr,
+  },
+  _createdAt,
+  "language": $language,
```

Comment:
i added this `"language": $language,` to the other queries just so i can do things like `category.name[category.language]`, so that I can use the name that was queried for in the first place (aka using the lang param passed in to the query). Just fyi